server/game/cards/12-KotI/GoldenStorm.js

```diff
@@ -0,0 +1,43 @@
+const DrawCard = require('../../drawcard.js');
+
+class GoldenStorm extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                afterChallenge: event => (
+                    event.challenge.winner === this.controller &&
+                    event.challenge.challengeType === 'military' && 
+                    event.challenge.attackingPlayer === this.controller                  
+                )
+            },
+            target: {
+                // activePromptTitle: 'Select a character',  // <- default prompt message, overridable
+                cardCondition: card => card.location === 'play area' && card.getType() === 'character' && card.isParticipating(),
+                gameAction: 'kill'                   
+            },
+            cost:  ability.costs.kneelSelf(),
+            handler: context => {
+                var wasStand = false;
+                context.target.owner.killCharacter(context.target);
+                let cards = this.controller.filterCardsInPlay(card => card.hasTrait('Raider') && card.getType() === 'character' && card.isParticipating() && card.controller === this.controller && card.location === 'play area');
+                if(cards.length > 0) {
+                    if(this.allowGameAction('stand')) {
+                        this.controller.standCard(this);
+                        wasStand = true;
+                    }
+                }
+                if(wasStand === true) {
+                    this.game.addMessage('[0] uses {1} to kill {2} and stand {3}',this.controller,this,context.target,this);
```

Comment:
Spaces after each comma would be more consistent with existing code style.

We should probably add that to the linting rules in a separate PR.